go.mod

```diff
@@ -25,7 +25,8 @@ require (
 	github.com/spf13/pflag v1.0.3
 	github.com/spf13/viper v1.3.1
 	github.com/vbauerster/mpb/v4 v4.8.4
-	golang.org/x/crypto v0.0.0-20191112222119-e1110fd1c708
+	github.com/vbauerster/mpb/v5 v5.0.4
```

Comment:
Is it *really* necessary to have both `vbauerster/mpb` v4 and v5? IMO you should either use v4 for this command, or upgrade the rest of the code to use v5 too.

Reply:
https://github.com/exoscale/cli/pull/249/commits/78d3bdd90981fda1c318c1818c526bfdb2a15da1